<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>CreateShortcutDialog</class>
 <widget class="QDialog" name="CreateShortcutDialog">
  <property name="windowModality">
   <enum>Qt::WindowModality::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>370</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Create Instance Shortcut</string>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="iconBtnLayout">
     <item>
      <widget class="QToolButton" name="iconButton">
       <property name="icon">
        <iconset>
         <normaloff>:/icons/instances/grass</normaloff>:/icons/instances/grass</iconset>
       </property>
       <property name="iconSize">
        <size>
         <width>80</width>
         <height>80</height>
        </size>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QGridLayout" name="iconBtnGridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="saveToLabel">
         <property name="text">
          <string>Save To:</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QComboBox" name="saveTargetSelectionBox">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
        </widget>
       </item>
       <item row="1" column="0">
        <widget class="QLabel" name="nameLabel">
         <property name="text">
          <string>Name:</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1">
        <widget class="QLineEdit" name="instNameTextBox">
         <property name="placeholderText">
          <string>Name</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <widget class="QCheckBox" name="overrideAccountCheckbox">
     <property name="toolTip">
      <string>Use a different account than the default specified.</string>
     </property>
     <property name="text">
      <string>Override the default account</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="accountOptionsGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QVBoxLayout" name="accountOptionsLayout">
      <item>
       <widget class="QComboBox" name="accountSelectionBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QCheckBox" name="targetCheckbox">
     <property name="toolTip">
      <string>Specify a world or server to automatically join on launch.</string>
     </property>
     <property name="text">
      <string>Select a target to join on launch</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="targetOptionsGroup">
     <property name="enabled">
      <bool>false</bool>
     </property>
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <layout class="QGridLayout" name="targetOptionsGridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="worldOverlap">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="worldTarget">
          <property name="text">
           <string>World:</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">targetBtnGroup</string>
          </attribute>
         </widget>
        </item>
       </layout>
      </item>
      <item row="0" column="1">
       <widget class="QComboBox" name="worldSelectionBox">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <layout class="QVBoxLayout" name="serverOverlap">
        <property name="spacing">
         <number>0</number>
        </property>
        <item>
         <widget class="QRadioButton" name="serverTarget">
          <property name="text">
           <string>Server Address:</string>
          </property>
          <attribute name="buttonGroup">
           <string notr="true">targetBtnGroup</string>
          </attribute>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="serverLabel">
          <property name="visible">
           <bool>false</bool>
          </property>
          <property name="text">
           <string>Server Address:</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="1">
       <widget class="QLineEdit" name="serverAddressBox">
        <property name="placeholderText">
         <string>Server Address</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="noteLabel">
     <property name="text">
      <string>Note: If a shortcut is moved after creation, it won't be deleted when deleting the instance.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QLabel" name="noteLabel2">
     <property name="text">
      <string>You'll need to delete them manually if that is the case.</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Orientation::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::StandardButton::Cancel|QDialogButtonBox::StandardButton::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>iconButton</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>CreateShortcutDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>CreateShortcutDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>20</x>
     <y>20</y>
    </hint>
    <hint type="destinationlabel">
     <x>20</x>
     <y>20</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <buttongroups>
  <buttongroup name="targetBtnGroup"/>
 </buttongroups>
</ui>
